cosmrs/src/crypto/public_key.rs

```diff
@@ -4,6 +4,7 @@ use crate::{prost_ext::MessageExt, proto, AccountId, Error, ErrorReport, Result}
 use eyre::WrapErr;
 use prost::Message;
 use prost_types::Any;
+use proto::cosmos;
```

Comment:
Personally I'd recommend not importing this and referring to it as `proto::cosmos::*` instead.

Inside of CosmRS it can get quite confusing whether you're referring to a protobuf type or a domain type since they have the same names, so using `proto::*` keeps it clear a type is a protobuf type.